js/index.js

```diff
@@ -1,7 +1,106 @@
+
+Quisque sed velit nec lorem vulputate volutpat non id massa. Aenean sed diam egestas ipsum dignissim facilisis eget eget libero. Etiam non erat massa. Proin bibendum, augue at fermentum finibus, metus odio molestie lectus, eu vulputate ipsum erat viverra tellus. Nam euismod vitae dolor sit amet facilisis. Sed nisi turpis, fringilla sed mi sed, cursus facilisis lectus. Suspendisse ut justo faucibus, faucibus sem eget, congue orci. Maecenas lobortis turpis libero, id venenatis elit porta pulvinar. Vivamus id ipsum id lorem vestibulum pulvinar. Nunc a magna placerat, viverra ex nec, mollis ante. Praesent pellentesque lectus at orci congue, nec viverra est suscipit. Duis in dictum augue. Aenean tincidunt luctus nunc eget tempus. 
+
+Sed laoreet ante vel quam auctor efficitur. Curabitur non metus vitae quam sollicitudin aliquet. Nunc imperdiet mi tortor, maximus sollicitudin urna volutpat et. Nunc eget nunc malesuada, ultrices tellus et, mollis orci. Pellentesque dapibus diam felis, sit amet consectetur lacus mollis at. Nunc gravida purus eget nibh bibendum mollis. Nulla ultricies metus ac turpis laoreet posuere. Proin eu dolor vel augue venenatis congue. Pellentesque convallis lorem id odio mattis, non sodales leo hendrerit. Nunc at malesuada metus.`
+
+
+function WordCount(str) { 
+  return str.split(" ").length;
+}
+
+console.log(WordCount(LoremStr));
+
+// Make your program count the number of times the Latin word [`et`] appears.
+
+function WordEtCount(str) { 
+    return str.split(`et`).length;
+  }
+  
+  console.log(WordEtCount(LoremStr));
+
+
+
+  /*#### Bonus 2:
+Create a new variable `phraseToCheck` and have it contain some string value. Write a code that will check if the value we assigned to this variable is a [Palindrome](https://en.wikipedia.org/wiki/Palindrome). Here are some examples of palindromes: 
+  - "A man, a plan, a canal, Panama!"
+  - "Amor, Roma"
+  - "race car"
+  - "stack cats"
+  - "step on no pets"
+  - "taco cat"
+  - "put it up"
+  - "Was it a car or a cat I saw?" and "No 'x' in Nixon".
+  
+  __Hint__: If you use Google to help you to find solution to this iteration, you might run into some solutions that use advanced string or array methods (such as _join()_, _reverse()_, etc.). However, try to apply the knowledge you currently have since you can build pretty nice solution with just using `for` loop, `if-else` statements with some `break` and `continue`... Just sayin' :smiley: */
+
+
+function palindrome(str) {
+
+    var newStr= str.toLowerCase().split("");
+  
+    for(var i=0; i < (newStr.length)/2; i++){ 
+      if(newStr[i] == newStr[newStr.length-i-1]){ 
+       
+      } else 
+        return false; 
+    }
+    return true;
+  }
```

Comment:
Please mind the indentation here.

Reply:
ok, tried fixing it. 
Still a bit confusing for me, but will get the grip of it eventually.